### ISSUE TITLE
NOTICE.md: Electron link outdated

### ISSUE THREAD

**Issue by nimo23** [eclipse-theia/theia#14501]
### Bug Description:

In https://github.com/eclipse-theia/theia/blob/master/NOTICE.md#electron, the link https://www.electronjs.org/docs/latest/development/upgrading-chromium#verify-ffmpeg-support returns `404 Not Found`.

**Comment by tsmaeder** [comment]
Thanks for reporting. @marcdumais since you last touched this file, do you know if there's some automation going into collecting the notices in this file?

**Comment by marcdumais-work** [comment]
> do you know if there's some automation going into collecting the notices in this file?

Yes, the rest of the file was auto-generated from the Theia project page, but the Electron notice was "hand-written" and provided by the Eclipse Foundation, following an investigation that we helped-with. I believe the referenced page, that no longer exists, contained an easy test - code to load in an Electron app to determine whether the version of libffmpeg bundled with an Electron app contains the allegedly proprietary codecs or not. 

I was able to find alternate sources for the page, however the link for the video at the heart of the test is itself no longer resolving. Look near the bottom of the page:
https://raw.githubusercontent.com/electron/electron/4305657858592be2b44c95ae7af53c627dcdc5e7/docs/development/upgrading-chromium.md

**Comment by per1234** [comment]
The link was fixed by https://github.com/eclipse-theia/theia/pull/15454


> the link for the video at the heart of the test is itself no longer resolving.

It is available from an archive of that URL here:

https://web.archive.org/web/20190518100139/http://www.quirksmode.org/html5/videos/big_buck_bunny.mp4
